particula/gas/properties/pressure_function.py

```diff
@@ -24,6 +24,14 @@ def calculate_partial_pressure(
     Returns:
     - float: Partial pressure of the gas in Pascals (Pa).
     """
+    # Input validation
+    if np.any(concentration <= 0):
```

Comment:
concentration can be zero, drop the =
```suggestion
    if np.any(concentration < 0):
```